src/bridge/jsHelpers.js

```diff
@@ -329,7 +334,7 @@ export const makeScanAccounts = (
                   : "no account"
               }`
             );
-            if (!account) return;
+            if (!account) break;
```

Comment:
imo we shouldn't change that. we should still return because the only case it's `!account` is when finished happened

Reply:
Ok, so it's ok if o.complete() is never called then ?

Reply:
if the consumer has unsubscribed, all observable events (next, error, complete) are ignored completely by rxjs.